library/connect.c

```diff
@@ -421,6 +427,10 @@ static int establish_crypto (nf_connection conn, message *msg) {
 
 static void flush_to_client(uv_async_t *fl) {
     nf_connection conn = fl->data;
+    if (conn == NULL || conn->state == Closed) {
```

Comment:
should this be after the log on line 424/434?

Reply:
nope -- can't check buffer if conn is NULL

Reply:
How about a different message then in this if?